.github/workflows/release.yml

```diff
@@ -6,7 +6,27 @@ on:
   workflow_dispatch:
 
 jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
```

Comment:
Is just python 3.10 supported by orca-recipes?

Reply:
We use the `apache/airflow:2.10.5-python3.10` base image in our Dockerfile.

Reply:
Nit: Should that info be broadcasted in the docs anywhere?  That we only support `python 3.10` even in our modules

Reply:
I think the development best practices around `src/`, `dags/` and `tests/` should be documented in CONTRIBUTING.md. I'll update.